yarn-project/pxe/src/contract_function_simulator/pxe_oracle_interface.ts

```diff
@@ -886,13 +886,25 @@ export class PXEOracleInterface implements ExecutionDataProvider {
     );
   }
 
-  public async removeNullifiedNotes(contractAddress: AztecAddress) {
+  /**
+   * Nullifies notes associated with a given contract that have been included onchain.
```

Comment:
```suggestion
   * Nullifies notes associated with a given contract.
```
Found the end of the sentence to be confusing and think it's better without it.